fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -669,7 +683,7 @@ public IndexInput openInput(@Nonnull final String name, @Nonnull final IOContext
         return new FDBIndexInput(name, this);
     }
 
-    public IndexInput openLazyInput(@Nonnull final String name, @Nonnull final IOContext ioContext, long initialOffset, long position) throws IOException {
```

Comment:
I'm assuming this was just removed because you noticed nothing was using it?

Reply:
There were FindBugs or PMD errors that mandated that I remove it all through the call chain (as the parameter wasn't used).